addons/purchase_requisition_stock/tests/test_purchase_requisition_stock.py

```diff
@@ -240,8 +240,6 @@ def test_04_purchase_requisition_stock(self):
         warning_wiz = Form(
             self.env['purchase.requisition.alternative.warning'].with_context(**action['context']))
         warning_wiz = warning_wiz.save()
-        self.assertEqual(warning_wiz.alternative_po_count, 1,
-                         "POs not in a RFQ status should not be listed as possible to cancel")
```

Comment:
> Just because you're removing the field doesn't mean this test isn't still valid. Adjust it accordingly:
> 
> ```
> self.assertEqual(len(warning_wiz.alternative_po_ids), 1,
>                  "POs not in a RFQ status should not be listed as possible to cancel")
> ```

I'd just add : "POs not in **an** RFQ...",
The sound of the following word ("are-ef-cue"), is what defines the "a/an".